.misc/deps.sh

```diff
@@ -8,12 +8,12 @@ fi
 
 # Choose the python versions to install deps for
 case $CIRCLE_NODE_INDEX in
```

Comment:
all of this circle stuff is going to get removed in the Circle CI patch, which is critical.
Lets get that in first.

Reply:
that is https://github.com/coala/coala/pull/5234

Reply:
merged.